src/language/typescript/common/data/serialized-type.ts

```diff
@@ -214,14 +214,16 @@ export const getSerializedOptionType = (serialized: SerializedType): SerializedT
 export const getSerializedOptionalType = (isRequired: boolean, serialized: SerializedType): SerializedType =>
 	isRequired ? serialized : getSerializedOptionType(serialized);
 
+export const getValidatedPropertyName = (name: string): string => (name.includes('-') ? `['${name}']` : name);
```

Comment:
Is this helper used anywhere else but in `getSerializedPropertyType`? If not then let's move it there.

Reply:
i mean, into the function?

Reply:
just inline it in `getSerializedPropertyType`

Reply:
ammended